data/data/install.openshift.io_installconfigs.yaml

```diff
@@ -1313,6 +1318,7 @@ spec:
                 - cloud
                 - computeFlavor
                 - externalNetwork
+                - ingressFloatingIP
                 - lbFloatingIP
```

Comment:
this should be renamed as APIFloatingIP then, right?

Reply:
Not sure we can simply rename it, we should instead deprecate `lbFloatingIP` in favor of `APIFloatingIP`.

Reply:
Yeah, I think that should really be a separate PR